vendor/modules.txt

```diff
@@ -39,7 +39,7 @@ github.com/opencontainers/selinux/pkg/pwalk
 github.com/pkg/errors
 # github.com/russross/blackfriday/v2 v2.0.1
 github.com/russross/blackfriday/v2
-# github.com/seccomp/libseccomp-golang v0.9.1
+# github.com/seccomp/libseccomp-golang v0.9.2-0.20210429002308-3879420cc921
```

Comment:
Can we have v0.9.2 soon?

Reply:
It's really not up to us :)

Reply:
I took a look at what it takes to have 0.9.2 and its milestone have quite a few opened issues (see https://github.com/seccomp/libseccomp-golang/milestone/3)

I tried to help a bit, creating a couple of PRs (https://github.com/seccomp/libseccomp-golang/pull/65, https://github.com/seccomp/libseccomp-golang/pull/66) and asked a question in https://github.com/seccomp/libseccomp-golang/issues/37#issuecomment-891488113. Alas, no reaction for the last 2 weeks :(

Reply:
In general, I think, looking into https://github.com/seccomp/libseccomp-golang/milestone/3 and trying to address some issues may help releasing 0.9.2.

Reply:
@kolyshkin nice, thanks! But AFAIK the lack of 0.9.2 doesn't limit this PR in any way, right? It will be nice to have 0.9.2, but AFAIK it won't happen anytime soon.

@AkihiroSuda also asked in this PR: https://github.com/seccomp/libseccomp-golang/pull/59#issuecomment-885756065, the response was to help with remaining items as you did :)